Added configure environment section...

eshanrnh opened the pull request:
## Description

... and replaced environment references

## Type of suggestion

* [x] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

Cloud

## Deadline (if relevant)

Anytime

sofietoft commented on umbraco-cloud/set-up/manage-environments.md:
```diff
@@ -1,35 +1,43 @@
-* For the Starter plan, you can add a Development environment for an additional price per month.
-* For the Standard plan, you get the Development environment for free and can add a Staging environment for an additional price per month.
-* For the Professional plan, you get the Development and Staging environment for free. Additionally, you can add and remove environments whenever you like without any additional cost.
+* **Starter plan:** Add a left-most mainline environment for an additional price per month.
+* **Standard plan:** The left-most mainline environment is included for free. Additional environments can be added for an additional price per month.
+* **Professional plan:** Mainline environments are included for free. You can add or remove environments at no extra cost.
```
I'd recommend that you copy paste the table that I've also added in several other articles, like Project Overview and Flexible Environments.
It's a little easier to read, and we ensure get the same information in all articles.
I'll look into adding is a a reusable block later on 😅

sofietoft commented on umbraco-cloud/set-up/manage-environments.md:
```diff
@@ -1,35 +1,43 @@
-**Important:** _Before_ adding an environment, you should consider if you have any changes locally that are not on Live yet. If you do, you should make sure to push it as adding another environment will also push it into the deployment chain.
+Before adding an environment, ensure there are no local changes that haven’t been pushed to Live. Adding an environment will push all changes in the current deployment chain.
+
+After adding a left-most mainline environment, you need to clone the site again. The local version will be set up to push directly to Live, while the fresh clone will push to the left-most mainline environment.
```
```suggestion
After adding a new left-most mainline environment or a flexible environment, you need to clone this environment instead. The current local clone will be set up to push to Live, while the fresh clone will push to the new environment.
```

sofietoft commented on umbraco-cloud/set-up/manage-environments.md:
```diff
@@ -1,35 +1,43 @@
+![Create environment](images/create-environment.png)
 
-Once you have added or removed an environment, it will take a couple of minutes for Cloud to set it all up, and then you will be ready to use it.
+3. Enter an **Environment name**.
```
To make it a little clearer that this is up to the user to decide.

```suggestion
3. Choose an **Environment name**.
```

sofietoft reviewed (commented):
I've added a couple of comments.

Instead of only having a heading for "Configuring an environment", wouldn't it make sense to have the following instead:

### Adding an environment
### Removing an environment

And these are added as subheadings to the ## Adding or Removing heading?

eshanrnh commented:
Thanks, @sofietoft for your review 🙌 I've incorporated your comments. Also, updated the Configuring Environments section to mention what actions are possible in that view.

sofietoft reviewed (commented):
Just one more thing, and this article is good to go 💪

eshanrnh commented on umbraco-cloud/set-up/manage-environments.md:
```diff
@@ -1,35 +1,50 @@
+It may take a few minutes for Cloud to set up the changes after adding or removing an environment.
 
-<figure><img src="../.gitbook/assets/image (17).png" alt=""><figcaption></figcaption></figure>
+## Configuring Environments
```
Reordered it above the Adding or Removing Environments section.

eshanrnh commented:
Thanks for the review, @sofietoft 🙌 I've incorporated your comments and fixed a broken image filename in the Environments article.
